### PR TITLE
[3.2.x] Improve integration tests to override endpoint security using the params file

### PR DESCRIPTION
**Pull request by wasuradananjith**
## Purpose
Fixes: https://github.com/wso2/product-apim-tooling/issues/769

## Goals
Integration tests didn't consider the endpoint security inside the endpoint config. This PR fixes it.

## Approach
- Consider endpoint security inside the endpoint config as well.
- Enhanced/Add the relevant tests for a super tenant and a tenant user with the Internal/devops role 
  - TestEnvironmentSpecificParamsEndpointSecurityDigestDevopsTenant
  - TestEnvironmentSpecificParamsEndpointSecurityDigestDevopsSuperTenant
  - TestEnvironmentSpecificParamsEndpointSecurityBasicDevopsTenant
  - TestEnvironmentSpecificParamsEndpointSecurityBasicDevopsSuperTenant
- Ran all the tests with wso2am-3.2.0+1626169644819.full.staging + the fix and all got passed.
  image

## Related PRs
- https://github.com/wso2-support/carbon-apimgt/pull/3569
- https://github.com/wso2-support/carbon-apimgt/pull/3570

## Test environment
- Ubuntu 20.04.2 LTS
- go version go1.16.3 linux/amd64

### CODE REVIEW

File: import-export-cli/integration/envSpecific_test.go [hunk 1]
```diff
@@ -149,9 +148,9 @@ func TestEnvironmentSpecificParamsEndpointSecurityFalse(t *testing.T) {
 	testutils.ValidateAPIsEqual(t, api, importedAPI)
 }
 
-func TestEnvironmentSpecificParamsEndpointSecurityDigest(t *testing.T) {
-	superTenantAdminUsername := superAdminUser
-	superTenantAdminPassword := superAdminPassword
+func TestEnvironmentSpecificParamsEndpointSecurityDigestDevopsSuperTenant(t *testing.T) {
```
Comment (Chamindu36) [inline import-export-cli/integration/envSpecific_test.go:151]: Shall we add a comment explaining the test case a bit as in other test cases[1]?

[1]. https://github.com/wso2/product-apim-tooling/blob/47ed2a862e5fc676c9f8ce801e885d9229fa0763/import-export-cli/integration/app_test.go#L484

File: import-export-cli/integration/envSpecific_test.go [hunk 2]
```diff
@@ -176,12 +175,42 @@ func TestEnvironmentSpecificParamsEndpointSecurityDigest(t *testing.T) {
...
 	apiParams := testutils.ReadAPIParams(t, args.ParamsFile)
 
-	validateEndpointSecurityDefinition(t, api, apiParams, importedAPI)
+	testutils.ValidateEndpointSecurityDefinition(t, api, apiParams, importedAPI)
 }
 
-func TestEnvironmentSpecificParamsEndpointSecurityBasic(t *testing.T) {
-	superTenantAdminUsername := superAdminUser
-	superTenantAdminPassword := superAdminPassword
+func TestEnvironmentSpecificParamsEndpointSecurityDigestDevopsTenant(t *testing.T) {
```
Comment (Chamindu36) [inline import-export-cli/integration/envSpecific_test.go:181]: Same here and please check the other places as well.

**Review by uvindra — APPROVED** [review]

File: import-export-cli/integration/envSpecific_test.go (hunk 1 above)
Comment (wasuradananjith) [reply import-export-cli/integration/envSpecific_test.go:159]: Fixed via e4f37dba86efef403f4a1d3967821d1f2791019b

File: import-export-cli/integration/envSpecific_test.go (hunk 2 above)
Comment (wasuradananjith) [reply import-export-cli/integration/envSpecific_test.go:191]: Fixed via e4f37dba86efef403f4a1d3967821d1f2791019b

**Review by Chamindu36 — APPROVED** [review]

**Pull request merged**